components/dss-tools/plugins/org.wso2.integrationstudio.artifact.dataservice/src/org/wso2/integrationstudio/artifact/dataservice/ui/wizard/DriverShim.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.integrationstudio.artifact.dataservice.ui.wizard;
+
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.DriverPropertyInfo;
+import java.sql.SQLException;
+import java.sql.SQLFeatureNotSupportedException;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+public class DriverShim implements Driver {
+    private Driver driver;
+
+    public DriverShim(Driver d) {
+        this.driver = d;
+    }
+
+    public boolean acceptsURL(String u) throws SQLException {
+        return this.driver.acceptsURL(u);
+    }
+
+    public Connection connect(String u, Properties p) throws SQLException {
+        return this.driver.connect(u, p);
+    }
+
+    public int getMajorVersion() {
+        return this.driver.getMajorVersion();
+    }
+
+    public int getMinorVersion() {
+        return this.driver.getMinorVersion();
+    }
+
+    public DriverPropertyInfo[] getPropertyInfo(String u, Properties p) throws SQLException {
+        return this.driver.getPropertyInfo(u, p);
+    }
+
+    public boolean jdbcCompliant() {
+        return this.driver.jdbcCompliant();
+    }
+
+    public Logger getParentLogger() throws SQLFeatureNotSupportedException {
+
+        return null;
+    }
+}
```

Comment:
Shall we add a new line here?

Reply:
Done https://github.com/wso2/integration-studio/pull/890/commits/7c67b03c80855728fbecedbfaad2e6f19af7f2c6